utils/build-script-impl

```diff
@@ -854,34 +854,19 @@ if [[ "${SKIP_RECONFIGURE}" ]]; then
     RECONFIGURE=""
 fi
 
-# WORKSPACE, BUILD_DIR and INSTALLABLE_PACKAGE must be absolute paths
-case "${WORKSPACE}" in
-    /*) ;;
-    *)
-        echo "workspace must be an absolute path (was '${WORKSPACE}')"
-        exit 1
-        ;;
-esac
-case "${BUILD_DIR}" in
-    /*) ;;
-    "")
-        echo "the --build-dir option is required"
-        usage
-        exit 1
-        ;;
-    *)
-        echo "build-dir must be an absolute path (was '${BUILD_DIR}')"
-        exit 1
-        ;;
-esac
-case "${INSTALLABLE_PACKAGE}" in
-    /*) ;;
-    "") ;;
-    *)
-        echo "installable-package must be an absolute path (was '${INSTALLABLE_PACKAGE}')"
-        exit 1
-        ;;
-esac
+# WORKSPACE, BUILD_DIR, and INSTALLABLE_PACKAGE must be absolute paths
+if ! [[ ${WORKSPACE} =~ ^/ ]] ; then
```

Comment:
Should we be quoting these variable expansions?

Reply:
No, it shouldn't be needed here.  The quoting behaviour of bash is far saner than that of POSIX sh.